dutree/dutree.py

```diff
@@ -468,34 +503,46 @@ def human(value):
 
 
 def main():
-    pathname = None
-    if len(sys.argv) == 2:
-        def getsize(node):
-            return node.app_size()
-
-        pathname = sys.argv[1]
-        use_apparent_size = True
+    parser = argparse.ArgumentParser(prog='dutree',
+                                     description='dutree shows a summary of \
+                                        the directories/files which take up\
+                                        the most space.')
+    parser.add_argument('--count-blocks', help='count block size not app size',
+                        action='store_true')
+    parser.add_argument('--xdev', help='stay on the same fileystem',
+                        action='store_true')
+    parser.add_argument('--no-skip-proc-sys',
+                        help='do not skip the filesystem devices\
+                            for proc and sys directories,\
+                            by default these are skipped',
+                        action='store_true')
```

Comment:
Yes, please reindent according to the 4-space indent (not the line-up indent).

Also:
_Backslashes at the end are only used in exceptional circumstances in python._

In the argparse case: all whitespace is collapsed anyway, so if you wanted to, you could do a multiline string with `"""`, or `'''`, but I prefer lining up several strings and being explicit about the whitespace.

Bad:
```python
help='do not skip the filesystem devices\
      for proc and sys directories,\
      ...'
```
Good:
```python
help=(
    'do not skip the filesystem devices '
    'for proc and sys directories, '
    ...
```

Because it is in fact:
```
do not skip the filesystem devices      for proc and sys directories,      ...
```
Instead of:
```
do not skip the filesystem devices for proc and sys directories, 
```

Reply:
i did not know that worked. new skill :heavy_check_mark: